docs/build-on-linea/linea-sdk/index.md

```diff
@@ -27,8 +27,44 @@ The SDK focuses on interacting with smart contracts on both Ethereum and Linea n
 
  5. Claiming messages (use one of the get message methods to grab all the parameters values)
 
+ 
+:::info[New Features]
 
-The code example below demonstrates how to initialize the SDK with the relevant configuration options, and how to call the featured methods.
+ The updated Linea SDK package enhances the L1 contract interaction and adds support for the new L1 claiming system, which is based on a merkle tree.
+ 
+ There are two important things to note:
+  - The previous L1 claiming and all function associated are still supported.
+  - The L2 claiming remains unaltered, and all SDK features to interact with L2 will remain unchanged.
+
+ The updated SDK introduces several new features for L1 interactions:
+
+ A new L1ClaimingService class that includes the following functions:
+  - getMessageProof: This function retrieves the message merkle tree proof required for new message claims on L1.
+  - isClaimingNeedingProof: This function determines whether a proof is needed to claim a message.
+  - getMessageStatus: This function retrieves a message's status, returning the status of both old and new messages.
+  - estimateClaimMessageGas: This function provides an estimate of the gas cost for both old and new claim transactions.
+  - claimMessage: This function enables a message to be claimed using either the old or new method.
+
+ Two new functions in the L1 contract:
+  - estimateClaimWithProofGas to estimates the gas cost for new claim transactions.
```

Comment:
`estimate` vs. `estimates`

Reply:
The formatting is generally inconsistent.

e.g. ``- getMessageProof:` has a colon, some places don't like in this one.

Some lines have periods. Some don't.

Some places we call it `functions` and some places `methods` - probably best to align on one.

Reply:
fixed